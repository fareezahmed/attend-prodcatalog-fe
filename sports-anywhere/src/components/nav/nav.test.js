import React from 'react';
import { shallow } from 'enzyme';

import Nav from './index';

import logo from '../../assets/logo.svg';

const setup = (partialProps) => {
    const props = {
        id: 1,
        title: "App Title",
        logo: logo,
        ...partialProps,
      }
    const wrapper = shallow(<Nav { ...props } />);
    return {
        wrapper,
        props,
    }
}

describe('Nav component', () => {
    it('component renders correctly', () => {
        const { wrapper } = setup();
        expect(wrapper).toMatchSnapshot();
    })
})