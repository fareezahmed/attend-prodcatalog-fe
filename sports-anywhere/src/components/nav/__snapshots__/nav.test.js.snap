// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nav component component renders correctly 1`] = `
<header
  className="nav"
>
  <Link
    replace={false}
    to="/"
  >
    <WithStyles(ForwardRef(IconButton))
      aria-label="menu"
    >
      <img
        alt="logo"
        className="App-logo"
        src="logo.svg"
      />
      <WithStyles(ForwardRef(Typography))
        variant="h6"
      >
        App Title
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(IconButton))>
  </Link>
</header>
`;
